source/confirm_conditions.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Confirm conditions
+weight: 50
+---
+
+# Confirm conditions
+Confirm that candidate has met conditions.
```

Comment:
I think for consistency, we should follow how we have already written these short descriptions e.g. see [retrieve a single application](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training-tech-docs/blob/master/source/retrieve-a_single-application.html.md.erb).

So:

- move the short description below the `POST /applications/:uuid/confirm_conditions` code block
- and with the same sentence structure / tense / language 👀 
```suggestion
Used to confirm that a candidate has met the conditions of an offer.
```

Tbh though, not sure how useful this sentence is. We could come to the conclusion that it's redundant. 🤔

Reply:
I agree, also I think it is fine to be explicit.